test/libsolidity/syntaxTests/emit/emit_non_event.sol

```diff
@@ -2,9 +2,8 @@ contract C {
   uint256 Test;
 
   function f() public {
-    emit Test();
+    emit f();
```

Comment:
This is not what the test is supposed to test...

Reply:
I thought the idea was to test the code that produces the error message "must emit event" and that one is only output if you use `f()`, if you use `Test()` you _only_ get "`Test is not callable`"

Reply:
Ah well, ok maybe it is, but then you can remove the ``uint256 Test;`` above.

Reply:
Or maybe best: change ``Test`` to be something callable.

Reply:
updated